content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -55,7 +55,7 @@ different Kubernetes components.
 | `APIResponseCompression` | `false` | Alpha | 1.7 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
 | `BalanceAttachedNodeVolumes` | `false` | Alpha | 1.11 | |
-| `BoundServiceAccountTokenVolume` | `false` | Alpha | 1.13 | |
+| `BoundServiceAccountTokenVolume` | `true` | Beta | 1.20 | |
```

Comment:
Instead of replacing the existing line, we should add a new line so the result would be:

```
| `BoundServiceAccountTokenVolume` | `false` | Alpha | 1.13 | 1.19 |
| `BoundServiceAccountTokenVolume` | `true` | Beta | 1.20 |  |
```